ma/config/magento.py

```diff
@@ -11,3 +11,40 @@
 
 _LOGGER.debug("CONFIG: S=[%s] U=[%s] P=[%s] H=[%s] P=(%d)",
               SCHEME, USERNAME, PASSWORD, HOSTNAME, PORT)
+
+# Hash used to execute remote re-indexing and cache refresh.
+UTILITY_HASH = os.environ.get('MAGENTO_UTILITY_HASH', None)
```

Comment:
You should put these in a separate config module that's specific to your new functional module so we don't ambiguously pool all of the config together. I recommend *ma.config.api.custom.maintenance* (to be completely formal, otherwise *ma.config.api.custom_maintenance*).